hm_25/hm_025.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" href="table.css">
+        <style>
```

Comment:
You have styles here and in the file you linked. It is better to keep styles in one place